### PR TITLE
testing/xfd: new aport

### PR DESCRIPTION
**Pull request by tanertas**
https://xorg.freedesktop.org/
Xorg font display application.

### CODE REVIEW

**Comment by tanertas** [comment]
@andypost can you take a look at this?

File: testing/xfd/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Contributor: Taner Tas <taner76@gmail.com>
+# Maintainer: Taner Tas <taner76@gmail.com>
+pkgname=xfd
+pkgver=1.1.2
+pkgrel=0
+pkgdesc="Xorg font display application."
+url="https://xorg.freedesktop.org/"
+arch="all"
+license="custom"
```
Comment (Ikke) [inline testing/xfd/APKBUILD]: This appears to use the [x11] license.

[x11]:https://spdx.org/licenses/X11.html

File: testing/xfd/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	libxaw-dev
+	libxft-dev
+	libxmu-dev
+	libxrender-dev
+	"
+subpackages="$pkgname-doc"
+source="
+	https://xorg.freedesktop.org/archive/individual/app/$pkgname-$pkgver.tar.gz
+	"
+#builddir="$srcdir/"
```
Comment (Ikke) [inline testing/xfd/APKBUILD]: This can be removed

File: testing/xfd/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+build() {
+	cd "$builddir"
+	export LIBS="$LIBS -lintl"
+	./configure --prefix=/usr
+	make
+}
+
+check() {
+	cd "$builddir"
+	make check
```
Comment (Ikke) [inline testing/xfd/APKBUILD]: It does not appear that make check does anything usefull:

> Making check in man
make[1]: Entering directory '/home/build/aports/testing/xfd/src/xfd-1.1.2/man'
make[1]: Nothing to be done for 'check'.
make[1]: Leaving directory '/home/build/aports/testing/xfd/src/xfd-1.1.2/man'
make[1]: Entering directory '/home/build/aports/testing/xfd/src/xfd-1.1.2'
make[1]: Leaving directory '/home/build/aports/testing/xfd/src/xfd-1.1.2'

If so, it's better remove it then giving the idea that this is executing a test suite.

File: testing/xfd/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+package() {
+	cd "$builddir"
+	make DESTDIR="$pkgdir" install
+	install -Dm644 COPYING "$pkgdir"/usr/share/licenses/$pkgname/COPYING
```
Comment (Ikke) [inline testing/xfd/APKBUILD]: If this is indeed using x11, than I guess this would not be necessary.

**Comment by tanertas** [comment]
@Ikke thanks for review. PR revised accordingly.

**Pull request closed without merging**